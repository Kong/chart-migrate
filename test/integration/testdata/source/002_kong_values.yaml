ingressController:
  enabled: true
  image:
    repository: kong/kubernetes-ingress-controller
    tag: "3.0"
  args:
  - --dump-config
  - "true"

  env:
    kong_admin_tls_skip_verify: true
    election_id: palmtree
  customEnv:
    TZ: "Europe/Berlin"
    FAKE: "whatever"

  livenessProbe:
    httpGet:
      path: "/healthz"
      port: 10254
      scheme: HTTP
    initialDelaySeconds: 5
    timeoutSeconds: 5
    periodSeconds: 10
    successThreshold: 1
    failureThreshold: 3
  readinessProbe:
    httpGet:
      path: "/readyz"
      port: 10254
      scheme: HTTP
    initialDelaySeconds: 5
    timeoutSeconds: 5
    periodSeconds: 10
    successThreshold: 1
    failureThreshold: 3
  resources:
    limits:
      cpu: 100m
      memory: 256Mi
    requests:
      cpu: 50m
      memory: 128Mi
