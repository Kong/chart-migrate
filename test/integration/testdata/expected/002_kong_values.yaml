ingressController:
  deployment:
    pod:
      container:
        args:
        - --dump-config
        - "true"
        customEnv:
          FAKE: whatever
          TZ: Europe/Berlin
        env:
          election_id: palmtree
          kong_admin_tls_skip_verify: true
        image:
          repository: kong/kubernetes-ingress-controller
          tag: "3.0"
        livenessProbe:
          failureThreshold: 3
          httpGet:
            path: /healthz
            port: 10254
            scheme: HTTP
          initialDelaySeconds: 5
          periodSeconds: 10
          successThreshold: 1
          timeoutSeconds: 5
        readinessProbe:
          failureThreshold: 3
          httpGet:
            path: /readyz
            port: 10254
            scheme: HTTP
          initialDelaySeconds: 5
          periodSeconds: 10
          successThreshold: 1
          timeoutSeconds: 5
        resources:
          limits:
            cpu: 100m
            memory: 256Mi
          requests:
            cpu: 50m
            memory: 128Mi
  enabled: true
